gaia-rs/src/message.rs

```diff
@@ -8,14 +8,14 @@ use gears::{
 };
 use serde::Serialize;
 
-#[derive(Debug, Clone, RoutingMessage, Serialize)]
+#[derive(Debug, Clone, AppMessage, Serialize)]
 #[serde(untagged)]
 pub enum Message {
-    #[gears(url = "/cosmos.bank.v1beta1")]
+    #[msg(url(string = "/cosmos.bank.v1beta1"))]
```

Comment:
It looks strange. Why did you use so complex syntax?

Reply:
To allow string and TypePath. Gaia part tries to parse any with prefix, while lower type by matching type(usually)